scripts/gke/deployment_manager_configs/cluster.jinja

```diff
@@ -21,7 +21,7 @@ limitations under the License.
    that will be created to represent Kubernetes objects.
    There is type corresponding to each API endpoint.
 #}
-{% set VM_OAUTH_SCOPES = ['https://www.googleapis.com/auth/logging.write', 'https://www.googleapis.com/auth/monitoring'] %}
+{% set VM_OAUTH_SCOPES = ['https://www.googleapis.com/auth/logging.write', 'https://www.googleapis.com/auth/monitoring', 'https://www.googleapis.com/auth/cloud-platform' ] %}
```

Comment:
If we add the third one, then first two is included.

@jlewi will it be too open to give anyone in cluster full access to all resources and services?

Reply:
Agree with Kunming why can't you use the namespace scoped secret.

Reply:
what's the risk of having cloud platform scope?